MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -55,15 +56,21 @@ def store_issue_details(issue, repo, db):
 	issue_details['created_at'] = issue['created_at']
 	issue_details['updated_at'] = issue['updated_at']
 	issue_details['labels'] = issue['labels']
-	
+
 	issue_details['issue_content_path'] = issue_content_path
 	# issue_details['test_output_path'] = test_output_path
 	issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+	issue_details['jenkins_output_path'] = jenkins_output_path
 	#Store issue content at issue_content_path
 	store_on_fs(issue['body'], issue_content_path)
+
 	store_quotation_result = store_on_fs(extract_quotation_content(issue['body']), issue_content_quotation_path)
 	if store_quotation_result:
 		issue_details['issue_content_quotation_path'] = issue_content_quotation_path
+
+	store_jenkins_result = store_on_fs(extract_jenkins_link_and_testname(issue['body']), jenkins_output_path)
```

Comment:
The content needs to query trss API before saving into fs, this can be added later after function `extract_jenkins_link_and_testname` worked.